preflight-checks/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "jest",
-    "build": "babel src --out-dir dist",
+    "build": "babel src --out-dir dist --copy-files --no-copy-ignored",
```

Comment:
What do these flags do?

Reply:
These flags make sure that the Dockerfile and the build.sh file (in the `src/docker`) directory are copied to the built application in `dist/`. 

https://babeljs.io/docs/en/babel-cli#copy-files